src/zap.conf

```diff
@@ -31,7 +31,7 @@
 10027	OUTOFSCOPE	http://app:8080/public/js/uswds-init.min.js
 # get-gov.js contains suspicious word "from" as in `Array.from()`
 10027	OUTOFSCOPE	http://app:8080/public/js/get-gov.js
-# Ignore wording of "TODO"
+# Ignores suspicious word "TODO"
```

Comment:
Ideally, for tracking purposes, you'd probably want to separate this out into its own PR, but since your branch was auto-deleted and being that its such a small comment change for something you intended to implement there, I think this is fine in here. I'd leave a comment somewhere on this PR when closing it out to [this](https://github.com/cisagov/getgov/pull/888#discussion_r1296029522) just for context if others need to get info on it

Reply:
Great call! It's the 4th bullet in the `changes` section, and will reference this file explicitly :)